4-float-features/4-float-features.md

```diff
@@ -0,0 +1,74 @@
+# Introduction
+
+Num type has been separated into Int type and Float type.\
+The old Num type operators and features only support Int types, so there is a need to support Float types also.
+
+We propose the addition of basic operators to Float types to match those of Int types.\
+Also, we propose more support in recognizing Int and Float types, such as operator type prediction.\
+When adding these features, consistency will be the key.
+
```

Comment:
Need a section on float syntax, mentioning the `_` support in particular. It should fully specify the syntax of floats in Hazel.